### DISCUSSION TITLE
WebRTC or similar function?

### DISCUSSION THREAD

**Discussion by dalisoft** [uNetworking/uWebSockets.js#216] [Q&A] [answered]
Hi @alexhultman 

Thanks for library, it's great and powerful (performance + stability).

It is possible send/receive videos like WebRTC?
Any demo or hints? With websocket possible, but that use a lot of CPU (80% CPU usage).

Any help appreciated.

Thanks

**Comment by ghost** [comment, marked as answer]
No that is not part of this project. You usually build WebRTC using websockets for signalling though, so they are not mutually exclusive. You still need some signalling server in any case.
